spectrocloud/resource_application_profile.go

```diff
@@ -105,6 +105,13 @@ func resourceApplicationProfile() *schema.Resource {
 							Description: "The name of the specified pack.",
 							Required:    true,
 						},
+						"properties": {
```

Comment:
@SivaanandM @nikchern we need to add the description fields to input attributes.

Reply:
Added description